en_ewt-ud-train.conllu

```diff
@@ -185640,7 +185640,7 @@
 12	.	.	PUNCT	.	_	5	punct	5:punct	_
 
 # sent_id = answers-20111107175720AAlb2TB_ans-0015
-# text = The Irish weather works on the ‘four seasons in a day’ principle, which basic­ally means that you can’t predict a thing when it comes to the behaviour of the sky.
+# text = The Irish weather works on the ‘four seasons in a day’ principle, which basically means that you can’t predict a thing when it comes to the behaviour of the sky.
```

Comment:
I think the word "basically" contains a soft hyphen. Are we supposed to remove such characters?

Reply:
These "soft hyphen" might break some programs, at least mine. I could not see them in my text editor.

Reply:
Per UniversalDependencies/UD_English-EWT#83, it sounds like we should keep the soft hyphen in the token. If you like you could add a `CorrectForm` feature that omits the soft hyphen.